### PR TITLE
Lab app integration

### PR DESCRIPTION
**Pull request by jordanhudgens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
 2. Map a route to a controller/action
-3. Render- implicit or explicit- a static template
+3. Render (implicitly or explicitly) a static template
 4. Create a static template
 
-## Notes
 
+## Testing
+
+The RSpec/Capybara test suite currently is not passing, to run the tests, run the command ```bundle exec rspec```, you will have passed the lab when the test is passing.
```
Comment (jmburges) [inline README.md]: By the time they get to this content, they will be very familiar with running tests! No need to include how to run them.

File: spec/features/hello_world_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+feature 'routing is working properly' do
+
+  scenario 'on the /hello_world page' do
+    visit hello_world_path
```
Comment (jmburges) [inline spec/features/hello_world_spec.rb]: We won't have covered `path`yet, so for this just do "/hello_world"

File: spec/features/hello_world_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+feature 'routing is working properly' do
```
Comment (jmburges) [inline spec/features/hello_world_spec.rb]: also, just for consistency can you do `describe` blocks instead of `feature` and `scenario`?

**Comment by jordanhudgens** [comment]
@AnnJohn I've integrated the changes and added meta data to master and solution branches.

**Pull request merged**